### PR TITLE
Add explainers to the different pages and update lookup title

### PR DESCRIPTION
**Pull request by ylembachar**

### CODE REVIEW

File: frontend/src/pages/Slot.tsx [hunk 1]
```diff
@@ -108,6 +108,12 @@ const Slot = () => {
         <ResponsiveLayout>
             <Box sx={{ flexGrow: 1, marginTop: 4 }}>
                 <TitleSection title="Slot Overview" />
+                <Typography variant="body1" textAlign="left" paddingTop="20px" color="#051016" lineHeight="180%">
+                    The slot overview is designed to give you information about the activity around the current blockchain's state.
+                    It gives a look ahead to the next 16 slots and shows when a registered validator is scheduled.
```
Comment (fredo) [inline frontend/src/pages/Slot.tsx:113]: It actually gives a look ahead to the next 16-32 slots

File: frontend/src/pages/TransactionLookup.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                            <b>Copy the Transaction Hash:</b> After submitting your transaction (e.g., via MetaMask), copy your
+                            transaction hash (Tx ID).
+                        </li>
+                        <li>
+                            <b>Paste in Shutter Explorer:</b> Go to the search bar above and paste your transaction hash into the input field.
+                        </li>
+                        <li>
+                            <b>View the Status:</b> The tool will show one of the following statuses:
+                            <ul>
+                            <li> Pending User Transaction: Your transaction has been sent but hasn’t been included yet.</li>
```
Comment (fredo) [inline frontend/src/pages/TransactionLookup.tsx:30]: Don't we also have the submitted state?

File: frontend/src/pages/Slot.tsx (hunk 1 above)
Comment (ylembachar) [reply frontend/src/pages/Slot.tsx:113]: For now, it's only the next 16 slots for the current epoch. I thought we could add the 32 when the next epochs is included?

Comment (fredo) [reply frontend/src/pages/Slot.tsx:113]: Ah makes sense. Yes

**Pull request merged**